src/components/peerstatsview/PeerStatsView.tsx

```diff
@@ -0,0 +1,203 @@
+import { InboundTrackEntry, OutboundTrackEntry } from '@observertc/client-monitor-js/lib/entries/StatsEntryInterfaces';
+import { StatsCollectedListener } from '@observertc/client-monitor-js/lib/EventsRelayer';
+import { useEffect, useState } from 'react';
+import { useContext } from 'react';
+import { ServiceContext } from '../../store/store';
+import Stats from './Stats';
+
+interface PeerStatsViewProps {
+	producerId?: string;
+	consumerId?: string;
+}
+
+type InboundStats = {
+	ssrc: number;
+	receivedKbps?: number;
+	fractionLoss?: number;
+}
+
+type OutboundStats = {
+	ssrc: number;
+	sendingKbps?: number;
+	RTT?: number;
+	Fps?: number;
+}
+
+function createOutboundStats(trackStats: OutboundTrackEntry): OutboundStats[] {
+	const result: OutboundStats[] = [];
+					
+	for (const outboundRtpEntry of trackStats.outboundRtps()) {
+		const remoteInboundRtpEntry = outboundRtpEntry.getRemoteInboundRtp();
+		const stats = outboundRtpEntry.stats;
+		const { bytesSent, timestamp } = outboundRtpEntry.appData?.traces || {};
+		const now = Date.now();
+		const elapsedTimeInMs = now - (timestamp ?? 0);
+		const dBytesSent = (stats.bytesSent ?? 0) - (bytesSent ?? 0);
+		const item: OutboundStats = {
+			ssrc: stats.ssrc,
+			sendingKbps: Math.floor(dBytesSent / (elapsedTimeInMs / 125)),
+			Fps: stats.framesPerSecond,
+			RTT: (remoteInboundRtpEntry?.stats.roundTripTime ?? 0) * 1000,
+		};
+		
+		result.push(item);
+
+		outboundRtpEntry.appData.traces = {
+			bytesSent: stats.bytesSent,
+			timestamp: now,
+		};
+	}
+
+	return result;
+}
+
+function createInboundStats(trackStats: InboundTrackEntry): InboundStats[] {
+	const result: InboundStats[] = [];
+
+	for (const inboundRtpEntry of trackStats.inboundRtps()) {
+		// inboundRtpStats.stats
+		const stats = inboundRtpEntry.stats;
+		const { 
+			packetsLost, 
+			packetsReceived, 
+			bytesReceived, 
+			timestamp } = inboundRtpEntry.appData?.traces || {};
+		const now = Date.now();
+		const elapsedTimeInMs = now - (timestamp ?? 0);
+		const dBytesReceived = (stats.bytesReceived ?? 0) - (bytesReceived ?? 0);
+		const dPacketsLost = (stats.packetsLost ?? 0) - (packetsLost ?? 0);
+		const dPacketsReceived = (stats.packetsReceived ?? 0) - packetsReceived;
+		const item: InboundStats = {
+			ssrc: stats.ssrc,
+			receivedKbps: Math.floor(dBytesReceived / (elapsedTimeInMs / 125)),
```

Comment:
Why `/ 125` here?

Reply:
`receivedKpbs` =  `receivedBytes / 1000 * 8` because 1 byte is 8 bit and I just simplified the `8/1000` part